assets/js/components/TargetIcon/TargetIcon.jsx

```diff
@@ -1,27 +1,41 @@
 import React from 'react';
 import { EOS_COLLOCATION, EOS_DESKTOP_WINDOWS } from 'eos-icons-react';
 
-import { TARGET_CLUSTER, TARGET_HOST } from '@lib/model';
+import { TARGET_CLUSTER, TARGET_HOST, isValidTargetType } from '@lib/model';
 
 const targetTypeToIcon = {
-  [TARGET_CLUSTER]: <EOS_COLLOCATION className="fill-white" />,
-  [TARGET_HOST]: <EOS_DESKTOP_WINDOWS className="fill-white" />,
+  [TARGET_CLUSTER]: EOS_COLLOCATION,
+  [TARGET_HOST]: EOS_DESKTOP_WINDOWS,
 };
 
-function TargetIcon({ targetType }) {
-  if (!targetType || !targetTypeToIcon[targetType]) {
+const targetTypeToLabel = {
+  [TARGET_CLUSTER]: 'Clusters',
+  [TARGET_HOST]: 'Hosts',
+};
+
+function TargetIcon({
+  targetType,
+  withLabel = false,
+  labelMap = targetTypeToLabel,
+  containerClassName = '',
+  iconClassName = '',
```

Comment:
You could just rename this as `className`

Reply:
and keep the other as `containerClassName`? I'd like to keep both of them